SPECS/python-wcwidth/python-wcwidth.spec

```diff
@@ -34,14 +45,21 @@ This Library is mainly for those implementing a Terminal Emulator, or programs t
 %py3_install
 
 %check
-%python3 setup.py test
+pip3 install \
```

Comment:
In my opinion, can we please double check the `%check` section, it seems it returns:

```bash
"====== CHECK DONE python-virtualenv. EXIT STATUS 4"
```

Reply:
fixed